distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Result_Set.enso

```diff
@@ -17,13 +17,13 @@ read_column result_set column_name =
     if result_set.isClosed then [] else
         index = result_set.findColumn column_name
 
-        builder = Vector.new_builder
-        go has_next = if has_next.not then Nothing else
-            builder.append (result_set.getObject index)
-            @Tail_Call go result_set.next
+        Vector.build builder->
+            go has_next = if has_next.not then Nothing else
+                builder.append (result_set.getObject index)
+                @Tail_Call go result_set.next
 
-        go result_set.next
-        builder.to_vector
+            go result_set.next
+            builder
```

Comment:
```suggestion
```

This line feels unnecessary.

The `Vector.build` discards the result returned from the lambda and returns the built vector anyway. So there is no need to return the `builder` from this lambda. I think we should simplify.